src/ILCompiler.MetadataWriter/src/ILCompiler.MetadataWriter.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I can add it, but it's dead weight right now because it's not used. I want to primarily use this from a unit test for now.

TL;DR: If you say you want it in the package now, I'll add it :).